fibbingnode/misc/mininetlib/iprouter.py

```diff
@@ -98,6 +99,22 @@ def __init__(self, router, *args, **kwargs):
         self.ospf.redistribute.static = 1000
         self.ospf.router_id = router.id
 
+        # Parse LSA throttling parameters
+        delay = CFG.get("DEFAULT", 'delay')
+        initial_holdtime = CFG.get("DEFAULT", 'initial_holdtime')
+        max_holdtime = CFG.get("DEFAULT", 'max_holdtime')
+
+        # Parse minimum LS intervals
+        min_ls_interval = CFG.get("DEFAULT", 'min_ls_interval')
+        min_ls_arrival = CFG.get("DEFAULT", 'min_ls_arrival')
+
+        self.ospf.throttling = {'spf': {'delay': delay,
```

Comment:
You should build these using chains of ConfigDict() dictionary types, e.g.
self.ospf.throttling = ConfigDict(spf=ConfigDict(delay=delay, initial_holdtime=...),
lsa-all=ConfigDict() ...)
(not sure you can have a - in key names that way)

This will simplify the config templates.